File: iOS/TFEAppDelegate.m
//
//  TFEAppDelegate.m
//  TwentyFortyEight-iOS
//
//  Created by Joshua Caswell on 9/29/16.
//  Copyright © 2016 Josh Caswell. All rights reserved.
//

#import "TFEAppDelegate.h"

#import "TFEGameController.h"

@interface TFEAppDelegate ()

@end

@implementation TFEAppDelegate

- (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
{
    UIWindow * window = [UIWindow new];
    TFEGameController * controller = [TFEGameController new];
    
    [self setMainWindow:window];
    [window setRootViewController:controller];
    
    [window makeKeyAndVisible];
    
    return YES;
}

@end

File: iOS/TFEAppDelegate.swift
//
//  TFEAppDelegate.swift
//  TwentyFortyEight
//
//  Created by Joshua Caswell on 10/1/16.
//  Copyright © 2016 Josh Caswell. All rights reserved.
//

import UIKit
import SpriteKit

@UIApplicationMain
class TFEAppDelegate : NSObject, UIApplicationDelegate
{
    var mainWindow: UIWindow!
    
    var gameController: TFEGameController!
    
    func application(application: UIApplication, didFinishLaunchingWithOptions launchOptions: [NSObject : AnyObject]?) -> Bool
    {
        let window = UIWindow()
        let controller = TFEGameController()

        self.mainWindow = window
        self.gameController = controller

        self.mainWindow.rootViewController = controller
        
        self.mainWindow.makeKeyAndVisible()
        
        return true
    }
}
